reference/5.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -456,6 +456,26 @@ Param(
 )
 ```
 
+### SupportsWildcards attribute
+
+The **SupportsWildcards** attribute is used to indicate that the parameter
+accepts wildcard values. The following example shows a parameter declaration
+the mandatory **Path** parameter that supports wildcard values.
```

Comment:
`a parameter declaration the mandatory`... missing word or combine sentences?  Looks like it needs a rephrase.

Reply:
fixed